Firestore/Example/Tests/Integration/FSTDatastoreTests.mm

```diff
@@ -153,6 +154,48 @@ - (void)rejectListenWithTargetID:(const TargetId)targetID error:(NSError *)error
 
 @end
 
+class RemoteStoreEventCapture : public RemoteStoreCallback {
+ public:
+  explicit RemoteStoreEventCapture(XCTestCase *test_case) {
+    underlying_capture_ = [[FSTRemoteStoreEventCapture alloc] initWithTestCase:test_case];
+  }
+
+  void ExpectWriteEvent(NSString *description) {
+    [underlying_capture_ expectWriteEventWithDescription:description];
+  }
+
+  void ExpectListenEvent(NSString *description) {
+    [underlying_capture_ expectListenEventWithDescription:description];
+  }
+
+  void ApplyRemoteEvent(const remote::RemoteEvent &remote_event) override {
```

Comment:
Nit: looks like you can just use `RemoteEvent` (there's a `using` declaration above). Applies throughout this file.